### PR TITLE
Powershell pastel colors support

### LINKED ISSUE #72: Pastel: Enabling colors in Windows Powershell.
Apparently Powershell has the ability to read ANSII escape characters for colors but it has the feature turned off by default. You can enable it by making a call to the windows API `SetConsoleMode()` in a small C stub. We could enable that only for windows perhaps.

### PR DESCRIPTION
**Pull request by saitonakamura**
I managed to cobble up some implementation and it's working, however I'm not sure about a couple things, since it's all new to me.

So here are several questions:
1. Is it a good idea to incude this right into `pastel` package? I've added this to `pastel-console` at first, but then realized that [hello-reason](https://github.com/esy-ocaml/hello-reason) uses `pastel` and sending it to right to `printf`.
2. What about the `js`? I've added a dummy js file and updated dune config as in `console/nativeChannels` but I'm hesitant in terms of if it's needed or not.
3. I'm not sure about marshaling error back to ocaml. I would say we don't care about the error in this case, but it's a subject to discuss

Fixes #72

### CODE REVIEW

**Comment by kyldvs** [comment]
Thanks for the contribution! I believe `pastel` is the correct place to add this. Will let @jordwalke respond to the other questions though.

File: src/pastel/EnableWinConsoleColors.re [hunk 1]
```diff
@@ -0,0 +1,8 @@
+external enableWinConsoleAniSeq: unit => unit = "enable_windows_console_ansi_seqences";
```
Comment (jordwalke) [inline src/pastel/EnableWinConsoleColors.re]: I think you meant for this to say `enableWinConsoleAnsiSeq` not `enableWinConsoleAniSeq`

Comment (saitonakamura) [reply src/pastel/EnableWinConsoleColors.re]: Fixed

**Comment by saitonakamura** [comment]
What about is js dummy file? Do I even need it?

**Comment by bandersongit** [comment]
> What about is js dummy file? Do I even need it?

I don't believe so (accordingly, I think you can also delete the JSOO stuff from the dune file)

**Comment by saitonakamura** [comment]
@jordwalke I believe it's ready to be merged

**Comment by saitonakamura** [comment]
Why yes, simple git revert did the trick :)

**Comment by bandersongit** [comment]
Thanks for doing this, merging.

**Comment by jordwalke** [comment]
Thank you @saitonakamura !

**Pull request closed without merging**